cross-chain/arbitrum/README.adoc

```diff
@@ -0,0 +1,42 @@
+:toc: macro
+
+= Threshold cross-chain - Arbitrum
+
+This package brings Bitcoin to Ethereum L2 Arbitrum. For more details please
+see link:https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-8.adoc[Cross-chain Tokenized Threshold BTC]
```

Comment:
```suggestion
see link:https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-8.adoc[RFC 8: Cross-chain Tokenized Threshold BTC]
```

Reply:
part of [d4c157b](https://github.com/keep-network/tbtc-v2/pull/554/commits/d4c157b18cc691fbd0b019a2495e6a66ea7557b2)